### ISSUE TITLE
运行 t_net 的时候参数具体是多少

### ISSUE THREAD

**Issue by Dugudaibo** [AITTSMD/MTCNN-Tensorflow#130]
在 ''gen_hard_example.py'' 中会调用
` t_net(args.prefix,#model param's file
          args.epoch, #final epoches
          args.batch_size, #test batch_size 
          args.test_mode,#test which model
          args.thresh, #cls threshold
          args.min_face, #min_face
          args.stride,#stride
          args.slide_window, 
          args.shuffle, 
          vis=False)`
但是这些参数具体是多少要从哪里看啊~

**Comment by Dugudaibo** [comment]
我好像自己看懂了一些，是不是在通过 p 网络产生 R 网络的困难样本时，应该将https://github.com/AITTSMD/MTCNN-Tensorflow/blob/b8631ba56ef5f3230f4c41358ef6dfcb94d114b9/prepare_data/gen_hard_example.py#L190的default='RNet'改为pnet，
然后将https://github.com/AITTSMD/MTCNN-Tensorflow/blob/b8631ba56ef5f3230f4c41358ef6dfcb94d114b9/prepare_data/gen_hard_example.py#L215的 net = 'ONet' 改为 Rnet

@AITTSMD

**Comment by AITTSMD** [comment]
@Dugudaibo 嗯，是的~

**Comment by homedawn** [comment]
def P_Net(inputs,label=None,bbox_target=None,landmark_target=None,training=True)的trainning的作用是否是：判定训练或测试模式，训练模型的时候要设置成“True”？？
